### DISCUSSION TITLE
Penpot on YUNOHOST. Thank you for supporting this campaign

### DISCUSSION THREAD

**Discussion by jrd10** [uxbox/uxbox#1309] [Show and tell]
Hi, this is José* from France.

I have taken the liberty to propose Penpot for YUNOSHOST.
YUNOHOST.org is a layer above Debian for easily install applications and services.  No need to be a sysadmin.

Here is a thread for launching the campaign :).
https://forum.yunohost.org/t/penpot-design-freedom-for-teams-on-yunohost-en-fr/17623
And a proposition to add it to the WhishList: https://yunohost.org/en/apps_wishlist

Thank in advance to devs who want to contribute :).
and all for your support :).

Thank you again for Penpot.

* Even if my first name sounds a little bit Spanish, I am French :).

**Comment by hirunatan** [comment]
Thanks José! It would be a good addition to the community!

At Penpot team currently we have not resources for packing the app anywhere than the official docker image, but if someone gets this task, they can ask us for support, of course.

**Comment by lapineige** [comment]
Is there any instructions for manual installation on Debian ? Even if it's not step-by-step a general guide would already be helpful for us to understand how to build a Yunohost package.

Thanks :)
